src/applications/personalization/view-dependents/layouts/ViewDependentsLayout.jsx

```diff
@@ -15,7 +16,9 @@ import { errorFragment, infoFragment } from './helpers';
 function ViewDependentsLayout(props) {
   let mainContent;
 
-  if (props.error && isServerError(props.error.code)) {
+  if (props.loading) {
```

Comment:
Unrelated to the work for this PR, but a quick add. This should fix our issue with an alertbox flashing when it shouldn't.